facebook-commerce.php

```diff
@@ -705,10 +722,13 @@ public function allow_full_batch_api_sync() {
 		 * @return boolean True if full batch sync is safe.
 		 * @since 2.6.1
 		 */
-		return apply_filters(
+		return apply_filters_deprecated(
 			'facebook_for_woocommerce_allow_full_batch_api_sync',
-			$default_allow_sync,
-			$this->get_product_count()
+			array(
+				$default_allow_sync,
+				$this->get_product_count(),
+			),
+			'x.x.x'
```

Comment:
This won't get auto replaced since it specifically looks for an `@since` tag, so some kind of note would be needed that this needs to be replaced manually during the release.

Reply:
true :/